src/main/java/org/launchcode/techjobs/oo/JobField.java

```diff
@@ -0,0 +1,55 @@
+package org.launchcode.techjobs.oo;
+
+import java.util.Objects;
+
+public abstract class JobField {
+
+    private int id;
+    private static int nextID = 1;
+    private String value = "Data not available";
```

Comment:
This isn't a required change, but I'd recommend giving this field a more specific name (ex. message, defaultMessage, etc.) just to ensure it provides details on what this field contains and what it will be used for.